qiskit_experiments/library/characterization/t2ramsey.py

```diff
@@ -25,33 +25,41 @@
 
 
 class T2Ramsey(BaseExperiment):
+    r"""T2Ramsey Experiment.
 
-    r"""
-    T2Ramsey Experiment
+    # section: overview
 
-    Overview
         This experiment is used to estimate two properties for a single qubit:
         T2* and Ramsey frequency.
+
+        See `Qiskit Textbook <https://qiskit.org/textbook/ch-quantum-hardware/\
+        calibrating-qubits-pulse.html>`_  for a more detailed explanation on
+        these properties.
+
         This experiment consists of a series of circuits of the form
 
-    .. parsed-literal::
+        .. parsed-literal::
+
+               ┌───┐┌──────────────┐┌──────┐ ░ ┌───┐ ░ ┌─┐
+          q_0: ┤ H ├┤   DELAY(t)   ├┤ P(λ) ├─░─┤ H ├─░─┤M├
+               └───┘└──────────────┘└──────┘ ░ └───┘ ░ └╥┘
+          c: 1/═════════════════════════════════════════╩═
+                                                        0
+
+        for each *t* from the specified delay times, and where
+        :math:`\lambda =2 \pi \times {osc\_freq}`,
+        and the delays are specified by the user.
+        The circuits are run on the device or on a simulator backend.
+        Results are analysed in the class T2RamseyAnalysis.
 
-           ┌───┐┌──────────────┐┌──────┐ ░ ┌───┐ ░ ┌─┐
-      q_0: ┤ H ├┤   DELAY(t)   ├┤ P(λ) ├─░─┤ H ├─░─┤M├
-           └───┘└──────────────┘└──────┘ ░ └───┘ ░ └╥┘
-      c: 1/═════════════════════════════════════════╩═
-                                                    0
+    # section: tutorial
+        :doc:`/tutorials/t2ramsey_characterization`
 
-    for each *t* from the specified delay times, and where
-    :math:`\lambda =2 \pi \times {osc\_freq}`,
-    and the delays are specified by the user.
-    The circuits are run on the device or on a simulator backend.
-    Results are analysed in the class T2RamseyAnalysis.
+    # section: analysis_class
 
-    Analysis Class
         :class:`~qiskit.experiments.characterization.T2RamseyAnalysis`
 
-    Analysis Options
+    # section: analysis_options
```

Comment:
Please move this to `T2RamseyAnalysis._default_options`. This is necessary to port documentation from parent class if experiment/analysis is nested and some one options are reused (Otherwise you need to manually manage arguments defined in the parent class).
```suggestion
    Analysis Options:
```

section syntax is not necessary except for the class documentation.